scaling_performance/optimizing_storage.adoc

```diff
@@ -87,8 +87,10 @@ The following table summarizes the recommended and configurable storage technolo
 | Yes
 | Configurable
 | Configurable
-| Configurable
-| Configurable
+a| Configurable footnoteref:[metrics-warning,For metrics it is an anti-pattern to use any shared storage and a single volume
+(RWX). By default, metrics deploys with one volume per Cassandra Replica.]
+a| Configurable footnoteref:[logging-warning,For logging, one volume per logging-es is required. Therefore using any shared
+storage would be an anti-pattern.]
```

Comment:
@bmcelvee Do you need the `a` before the row? The footnote below does not use the `a`. 
Also, it seems a bit awkward that the first footnote starts with the anti-pattern statement then the OpenShift configuration; but, the second is reversed. 
s/For metrics/For metrics, (add a comma for consistency between the two.

Reply:
Thanks, @mburke5678! I reordered the statements to make them line up better. For the`a`, I had several ci-test failures and put it in as an effort to resolve them. All fixed!